Apps/WebClient/src/Server/Startup.cs

```diff
@@ -168,7 +170,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                         new SpaOptions { SourcePath = "ClientApp" },
                         npmScript: "serve",
                         port: 8585,
-                        regex: "Compiled successfully",
+                        regex: "Compiled ",
```

Comment:
should we leave the "successfully" here and wait for the fix?

Reply:
not necessary, if it compiles it should work ok, even with warnings